server/src/main/java/org/cloudfoundry/identity/uaa/login/LoginInfoEndpoint.java

```diff
@@ -361,6 +361,21 @@ private String login(Model model, Principal principal, List<String> excludedProm
             returnLoginPrompts = false;
         }
 
+        if (jsonResponse && request.getRequestURI().endsWith("/info")) {
+            List<Map<String, String>> infoIdentityProviders = new java.util.ArrayList<>();
+            providerProvisioning.retrieveAll(true, IdentityZoneHolder.get().getId())
```

Comment:
We have tweaked this endpoint the last years not to retrrieve always all IdP because we (SAP) have many IdPs in one zone and with cf login there is the situation, that /info is called before the login and with that we had DB issues

@torsten-sap correct or is my assumption no longer valid ?

Reply:
Sounds like a property to explicit enable this feature is needed, with the default being set to `false`

Reply:
problem is that we had many changes in this area, some because to fix DB performance, and later some to repair some regressions, we should discuss in meeting